src/middlewares/auth.middleware.ts

```diff
@@ -50,6 +60,30 @@ export const restrictTo = (...roles: any) => {
   };
 };
 
+/* export const verifyToken = (req: Request, res: Response, next: NextFunction) => {
+  const authHeader = req.header('Authorization');
+  const token = authHeader && authHeader.split(' ')[1];
+
+  if (!token) {
+    return res.status(401).json({
+      status: "error",
+      message: "Access denied. No token provided.",
+    });
+  }
+
+  try {
+    const decoded = jwt.verify(token, process.env.JWT_SECRET || 'default_secret');
+    req.user = decoded;
+    next();
+  } catch (error) {
+    console.error("Invalid token:", error);
+    return res.status(401).json({
+      status: "error",
+      message: "Invalid token.",
```

Comment:
Thank you for your work but Please remove all the commented codes.

Reply:
Removed